tests/complete/functions.py

```diff
@@ -28,6 +28,9 @@ def circle_lines(s_x, s_y, height):
 
 
 def fib(n: int):
+    """
```

Comment:
What's the point of this? Or will the comment be more descriptive in the future?

Reply:
Test to make sure the added feature works.

Users can not have triple-quote comments in the functions they decorate with `verilogify`.